tox.ini

```diff
@@ -14,6 +14,7 @@ deps =
   coverage
   pytest
   pytest-cov
+  typing; python_version < "3.0"
```

Comment:
I find type-hints already quite usable since Python 3.4 (I have been using the extensively ever since). But I'm not hung up on them. Especially as vulture is a CLI tool rather than a library. IMO they make most sense on the external "contact point" of an API/library. Removed in f272778.